speech/cloud-client/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
Google LLC

Reply:
If this throws a style issue. May need to update the pom.xml line 29 to `<version>1.0.11</version>`? (or maybe just 10)